<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>京剧</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rulesButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>关于作者</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../venv/Lib/site-packages/QtDesigner/bckg.qrc"/>
 </resources>
 <connections/>
</ui>
